packages/arb-ts/src/lib/bridge.ts

```diff
@@ -850,4 +858,40 @@ export class Bridge {
       this.l2Provider
     )
   }
+
+  public async getTokenBalanceBatch(
+    userAddr: string,
+    tokenAddrs: Array<string>,
+    targetNetwork: 'L1' | 'L2'
+  ) {
```

Comment:
Can we add typing to the return value here

Reply:
This was added in the follow up PR
https://github.com/OffchainLabs/arbitrum/pull/1673

Reply:
https://github.com/OffchainLabs/arbitrum/pull/1673/commits/62248797b99b70552b0a24f93bc9bb4372140a48